### ISSUE TITLE
Integrate ZORB as an opt-in optimization

### ISSUE THREAD

**Issue by LifeIsStrange** [microsoft/fastformers#9]
Fastformers is already impressive as is but a new paper has just been released: ZORB 
And it aims to become an alternative to... Backpropagation (you read this right) 

Such a breakthrough allow ~300X additional speedup to fastformers (even more in theory) while only diminishing accuracy of a few percents on many cases and actually apparently outperforming BP with a lower error count? 
Anyway it's a huge performance breakthrough and should be popularized by fastformers and huggingface. 

https://paperswithcode.com/paper/zorb-a-derivative-free-backpropagation
Notes: 
Extensive testing of the variable accuracy loss would be welcome. 
Some activations functions needs to be adapted? (e.g Mish?)

**Comment by ykim362** [comment]
Hi @LifeIsStrange .
The paper you pointed is about faster model training. And, the authors proved it's working on CNN.
On the other hand, FastFormers is for faster inference.
I haven't read the paper fully, but it might be cool if it also works for transformer architectures.
Thanks.

**Comment by LifeIsStrange** [comment]
@ykim362 oh OK if training optimizations is out of scope for fastformers, feel free to close this issue :)

**Comment by ykim362** [comment]
Yes, training is out of scope for this repository. We have a separate project for that (not opened yet). Thanks!
